### ISSUE TITLE
Request: Kinetic Whip

### ISSUE THREAD

**Issue by LadyAlekto** [Vek17/WrathMods-TabletopTweaks#111]
I do not know why Owlcat removed it, but kingmaker had it

https://www.d20pfsrd.com/alternative-rule-systems/occult-adventures/occult-classes/kineticist/archetypes/paizo-llc-kineticist-archetypes/kinetic-knight-kineticist-archetype/

**Comment by Vek17** [comment]
Pretty sure this was mod only in KM iirc

**Comment by Mira132** [comment]
Yeah, this was only available through a mod in kingmaker, so Owlcat didn't remove it, but they really should add it because melee kineticist really sucks without it thanks to not being able to do attacks of opportunities with kinetic blade. Though that's just one of the many things missing with their kinetic knight implementation...

**Comment by LadyAlekto** [comment]
Actually reinstalled KM and its correct, CotW added the whip and many missing kineticist features, like Kinetic Fist or Kinetic Invocation

Wondering also how hard would it be to add Elemental Annihilator?

Basically using blasts for multiple attacks instead of all or nothing
